libclamav/ole2_extract.c

```diff
@@ -256,8 +256,8 @@ cli_ole2_get_property_name2(const char *name, int size)
     if ((name[0] == 0 && name[1] == 0) || size <= 0 || size > 128) {
         return NULL;
     }
-    CLI_MAX_MALLOC_OR_GOTO_DONE(newname, size * 7,
-                                cli_errmsg("OLE2 [cli_ole2_get_property_name2]: Unable to allocate memory for newname: %u\n", size * 7));
+    CLI_MAX_MALLOC_OR_GOTO_DONE(newname, size * 5,
```

Comment:
Could you add a comment specifying the significance of the 5?

Reply:
Good point.  I will do so or else make a #define with a descriptive name.

Reply:
Fixed.